subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/results/IndexPageGenerator.java

```diff
@@ -185,9 +186,16 @@ protected String determineScenarioBackgroundColorCss(ScenarioBuildResultData sce
                 if (scenario.isFromCache()) {
                     result.add(FROM_CACHE);
                 }
+
                 if (isFlaky(scenario)) {
                     result.add(FLAKY);
+                } else {
```

Comment:
Why are we doing this in the `else` block? If the scenario is flaky and I find a known issue, it would be great to have a link as well.

Reply:
Yes, we can do that.

Reply:
Done.